app/grandchallenge/cases/models.py

```diff
@@ -322,6 +344,12 @@ def shape_without_color(self) -> List[int]:
         result.append(self.width)
         return result
 
+    def full_clean(self):
+        if self.segments is not None:
+            # panimg returns a tuple, which does not play nicely with jsonschema
```

Comment:
That is what I initially had there, but because the `PanImg` class in panimg is a `dataclass` its fields need to be hashable. Apparently we could work around that by either implementing our own `__hash__` method or by excluding non hashable fields from the hash, but this seemed like a better solution to me. (However, I did not realize using a tuple would give issues with validation in gc.)